src/Containers/App/index.jsx

```diff
@@ -777,7 +777,36 @@ class App extends Component {
     }
 
     try {
-      const apiVersion = await this.serial.getApiVersion();
+      const apiVersion = await this.serial.getApiVersion().catch(async (e) => {
```

Comment:
Please use async/await instead of chaining. You can catch for the explicit error:

```javascript
try {
 // const apiVersion = ...
} catch(e) {
  if(e instanceof TimeoutError) {
    // Your error handling
  } else {
    // Generic exception handling
  }
}
```

Reply:
done